master/pkg/searcher/operations.go

```diff
@@ -28,11 +28,10 @@ type (
 
 // All the operation types that support serialization.
 const (
-	CreateOperation OperationType = iota
-	TrainOperation
-	ValidateOperation
-	CheckpointOperation
-	CloseOperation
+	CreateOperation   OperationType = 0
```

Comment:
iotas are bad and lead to bugs, i'm totally convinced now (in some specific cases, or if you're an idiot like me)